src/sentry/issues/auto_source_code_config/task.py

```diff
@@ -161,23 +166,25 @@ def get_trees_for_org(
         return trees
 
 
-def create_repos_and_code_mappings(
+def create_configurations(
```

Comment:
We will be creating more than repositories and code mappings, thus, a more generic name.